src/components/SplashScreen.tsx

```diff
@@ -0,0 +1,25 @@
+import Circle from "./splashScreen/Circle";
+import "./splashScreen/style/SplashScreen.css";
```

Comment:
**nitpick:** We should keep components styles close to it's implementation. It will highly improve your codebase maintainability. Eg:

```ts
import styles from "./SplashScreen.module.css";
```

Reply:
Thanks for guidance, had a dillemma, where to put this file :')